### PR TITLE
PCHR-1522: Views permissions

### PR DESCRIPTION
**Pull request by coldrunKacper**
Updating Feature module with permissions requirements for Reports and Calendar Views.
Added 'Access HR Reports' required permission for:
- calendar_absence_list View and all its Displays
- civihr_report_people and all its Displays
- civihr_report_leave_and_absence View and all its Displays

Included Calendar view into Features.

### CODE REVIEW

**Comment by AkA84** [comment]
Approved

File: civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.views_default.inc [hunk 1]
```diff
@@ -1968,5 +1968,409 @@ return date(\'Y-m-d\');';
   );
   $export['civihr_report_leave_and_absence'] = $view;
 
+  $view = new view();
+  $view->name = 'calendar_absence_list';
```
Comment (AkA84) [inline civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.views_default.inc]: was this view previously included somewhere else?

Comment (coldrunKacper) [inline civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.views_default.inc]: This view is included in SSP module and is installed with SSP module installation but was not included into Features so I've added it to be able to update its permissions by doing features-revert.

**Comment by AkA84** [comment]
Approved

**Pull request merged**